### PR TITLE
Init hibernate entities

### PR DESCRIPTION
**Pull request by mukasz**

### CODE REVIEW

**Comment by mukasz** [comment]
Please checkout and review packages structure before approval
@PrzemyslawZnojek  @Kusibab

File: weddineo/src/main/java/com/weddineo/checklist/dao/wedding/WeddingOrganizerRepository.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.weddineo.checklist.dao.wedding;
+
+import com.weddineo.checklist.model.wedding.WeddingOrganizer;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface WeddingOrganizerRepository
```
Comment (Kusibab) [inline weddineo/src/main/java/com/weddineo/checklist/dao/wedding/WeddingOrganizerRepository.java]: missing @repository

File: weddineo/src/main/java/com/weddineo/firebase/service/FirebaseAppInitializer.java [hunk 2]
```diff
@@ -25,17 +25,17 @@
...
-            if (firebaseAppIsEmpty()) {
-                initializeFirebaseApp();
-            }
-        } catch (FileNotFoundException e) {
-            throw new WeddineoFirebaseRuntimeException("Firebase credentials file not found");
-        } catch (IOException e) {
-            throw new WeddineoFirebaseRuntimeException("Cannot read firebase credentials file");
-        }
-        log.info("Firebase service initialized successfully");
+//        log.info("Initializing firebase service...");
```
Comment (Kusibab) [inline weddineo/src/main/java/com/weddineo/firebase/service/FirebaseAppInitializer.java]: commented code

File: weddineo/src/main/resources/application.yml [hunk 3]
```diff
@@ -1,10 +1,25 @@
 spring:
   profiles:
     active: '@spring.profiles.active@'
+  datasource:
+    url: jdbc:postgresql://192.168.99.100:5432/weddineo
+    username: postgres
```
Comment (Kusibab) [inline weddineo/src/main/resources/application.yml:6]: should we store this in here?

**Review by PrzemyslawZnojek — COMMENTED** [review]
Please uncomment code in FirebaseAppInitializer.java

File: weddineo/src/main/resources/application.yml (hunk 3 above)
Comment (mukasz) [reply weddineo/src/main/resources/application.yml:6]: Yes, until this properties are for developnent usage datasource config could be stored here in plain text(it refers to local database) - in future we could think about externalizing properties and attaching them during application launch.

I will replace IP in datasource.url to (...)localhost:5432(...) <- it was my mistake to commit sppecified addres :)

**Comment by mukasz** [comment]
> 
> 
> Please uncomment code in FirebaseAppInitializer.java

done

File: weddineo/src/main/java/com/weddineo/firebase/service/FirebaseAppInitializer.java (hunk 2 above)
Comment (mukasz) [reply weddineo/src/main/java/com/weddineo/firebase/service/FirebaseAppInitializer.java]: done

**Comment by mukasz** [comment]
> 
> 
> Please uncomment code in FirebaseAppInitializer.java

done

File: weddineo/src/main/java/com/weddineo/checklist/dao/wedding/WeddingOrganizerRepository.java (hunk 1 above)
Comment (mukasz) [reply weddineo/src/main/java/com/weddineo/checklist/dao/wedding/WeddingOrganizerRepository.java]: done

**Review by PrzemyslawZnojek — APPROVED** [review]

**Review by Kusibab — APPROVED** [review]

**Pull request merged**